src/ast/ast_rewrite_same_clauses.c

```diff
@@ -334,13 +292,11 @@ static inline bool is_compressible
 ) {
 	// compressible clauses:
 	// 1. Non-OPTIONAL MATCH
-	// 2. CREATE
-	// 3. SET
-	// 4. DELETE
-	// 5. REMOVE
+	// 2. SET
```

Comment:
maybe we need to introduce an optimization that merge this if we can?

Reply:
no need I believe we can remove it.